classes/Language.php

```diff
@@ -707,14 +707,34 @@ public static function getLangDetails($iso)
     public static function getLocaleByIso($isoCode)
     {
         if (!Validate::isLanguageIsoCode($isoCode)) {
-            throw new Exception(sprintf('The ISO code %s is invalid'));
+            throw new Exception(sprintf('The ISO code %s is invalid', $isoCode));
```

Comment:
Actually, if we don't plan to translate it, we can remove the sprintf() which is useless.